integrations/react.md

```diff
@@ -3,107 +3,151 @@ layout: default
 title: React integration
 title_nav: React
 description: React TinyMCE component.
-keywords: integration integrate react reactjs create-react-app
+keywords: integration integrate react reactjs create-react-app tinymce-react
 ---
 
-Integration with React is easily done with our [@tinymce/tinymce-react](https://github.com/tinymce/tinymce-react) package. This how-to shows you how to setup a project using [react](https://facebook.github.io/react/), [tinymce](/docs/demo/basic-example/) and [Create React App](https://github.com/facebookincubator/create-react-app).
+The [Official TinyMCE React component](https://github.com/tinymce/tinymce-react) integrates TinyMCE into React projects.
+This procedure creates a [basic react application](https://github.com/facebook/create-react-app) containing a TinyMCE editor based on our [Basic example]({{site.baseurl}}/docs/demo/basic-example/).
+
+For examples of the TinyMCE integration, visit [the tinymce-react storybook](https://tinymce.github.io/tinymce-react/).
+
+## Prerequisites
+
+This procedure requires:
+* [Node.js (and npm)](https://nodejs.org/).
+* Access to `tinymce.min.js` on:
```

Comment:
Do we need something like `either` added here? As atm it sounds like you need access to both to be able to use the react intergration.

Reply:
adding `either`